### PR TITLE
[WNMGDS-1592] storybook example

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

* Pulled CodeSnippet code into reusable component and added it to all examples (Responsive, Embedded & Storybook)
* Created a reusable `StorybookExample` component that shows an iframe to the storybook story (for components) and includes a CodeSnippet
  * This component has some similarities from ResponsiveExample for some iframe responsiveness, but since there's no breakpoints, the markup has to be structured a little differently
* Updated comment around components for some clarity

## How to test

`yarn build-storybook:gatsby` to build storybook once
`yarn build:gatsby && yarn serve:gatsby` to spin up the doc site

Screen Shot 2022-05-16 at 10 50 01 AM

### CODE REVIEW

File: packages/docs/src/components/ResponsiveExample.tsx [hunk 1]
```diff
@@ -78,54 +84,62 @@ const ResponsiveExample = ({ storyId, title }: ResponsiveExample) => {
       contentWindow.addEventListener('resize', handleIframeResize);
       iframeRef.current.contentDocument.documentElement.classList.add('ds-u-overflow--hidden');
       handleIframeResize();
+
+      const rootEl = iframeRef.current.contentDocument.body.querySelector('#root');
+      if (rootEl) {
+        setiFrameHtml(rootEl.innerHTML);
+      }
```
Comment (pwolfert) [inline packages/docs/src/components/ResponsiveExample.tsx:91]: Cool!

File: packages/docs/content/5_components/alert.mdx [hunk 2]
```diff
@@ -6,12 +6,16 @@ Alerts keep users informed of important and sometimes time-sensitive changes.
 
 ## `<Alert>`
 
+<StorybookExample componentName="alert" sourceFilePath="/components/Alert/Alert.tsx" />
```
Comment (pwolfert) [inline packages/docs/content/5_components/alert.mdx:9]: I just noticed that this turned into `https://github.com/CMSgov/design-system/blob/master/packages/design-system/src//components/Alert/Alert.tsx` with `...src//components...`

**Review by pwolfert — APPROVED** [review]
This is great!

**Comment by line47** [comment]
@scheul93  think it would be nice to right align the link out to storybook like on the current doc site does for open in new window

**Comment by scheul93** [comment]
Updated the 'open in storybook' link
Screen Shot 2022-05-16 at 4 54 33 PM

File: packages/docs/content/5_components/alert.mdx (hunk 2 above)
Comment (scheul93) [reply packages/docs/content/5_components/alert.mdx]: Fixed!

**Pull request merged**